data_models/bioinformatics_mill_models/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.utils import timezone
+
+
+class TimeTrackedModel(models.Model):
+    created_at = models.DateTimeField(editable=False)
+    updated_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        ''' On save, update timestamps '''
+        if not self.id:
+            self.created_at = timezone.now()
+        self.updated_at = timezone.now()
```

Comment:
Interesting. The documentation at https://docs.djangoproject.com/en/1.10/topics/i18n/timezones/ is confusing. Near the top, it says

> Time zone support is disabled by default. To enable it, set USE_TZ = True in your settings file. 

Then, in the first note it says

> The default settings.py file created by django-admin startproject includes USE_TZ = True for convenience.

I guess that's actually consistent, but why should `startproject` second-guess the decision that "time zone support is disabled by default?"